.gitpack/data/asus-fan-control.service

```diff
@@ -12,3 +12,4 @@ ExecStart=/usr/local/bin/asus-fan-control
 
 [Install]
 WantedBy=multi-user.target
+Also=asus-fan-control-resume.service
```

Comment:
I trust you with this as I do not know the syntax and the real importance of it... :smile:

Reply:
It enables `asus-fan-control-resume` when `asus-fan-control` is getting enabled
https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Also=

Reply:
Like when you run `systemctl enable asus-fan-control` it also does what `systemctl enable asus-fan-control-resume` would do.

Reply:
Great! :rocket: